src/components/policy/policy_external/src/policy_table/types.cc

```diff
@@ -665,6 +667,9 @@ bool ModuleConfig::is_valid() const {
   if (!preloaded_date.is_valid()) {
     return false;
   }
+  if (!lock_screen_dismissal_enabled.is_valid()) {
+    return false;
+  }
```

Comment:
@LuxoftAKutsan Check `IF` statements indents.
May we add some short logs to every `IF` statement to get and idea what's going on?

Reply:
No logger is available in this level.